packages/web3-utils/package-lock.json

```diff
@@ -1,8 +1,6 @@
 {
-	"name": "web3-utils",
-	"version": "1.0.0-beta.41",
```

Comment:
Its a little concerning that the lockfile would change like it does here without any corresponding changes to package.json, etc... Any idea what'd going on here?

Reply:
This is because I've done a clean install and deleted all the package-lock.json files before. Will finish this work and release it ~beta.45